doc/api/zlib.md

```diff
@@ -4,60 +4,124 @@
 
 > Stability: 2 - Stable
 
-The `zlib` module provides compression functionality implemented using Gzip and
-Deflate/Inflate, as well as Brotli. It can be accessed using:
+The `zlib` module provides compression functionality implemented using Gzip,
+Deflate/Inflate, and Brotli.
+
+It can be accessed using:
```

Comment:
FWIW, we use this in some of our docs:

```suggestion
To access it:
```

I prefer it because it is active voice, direct, and concise. But admittedly, this is Just My Opinion.

Reply:
^^^^ Non-blocking, obviously. I also like  `Access it with:` if that's more agreeable for others.